packages/hash/api/src/graphql/typeDefs/comment.typedef.ts

```diff
@@ -0,0 +1,99 @@
+import { gql } from "apollo-server-express";
+
+export const commentTypedef = gql`
+  type Comment implements Entity {
+    properties: CommentProperties!
+
+    contents: [TextToken!]!
+
+    owner: User!
+
+    parent: Block!
```

Comment:
Nit: these fields are missing documentation.

Reply:
Added in 271ed0520178e75fb4def14d6878512b9fde0f11.